docs/rst/user_manual/configuration.rst

```diff
@@ -10,20 +10,27 @@ A |ddsrouter| is configured by a *.yaml* configuration file.
 This *.yaml* file contains all the information regarding the |ddsrouter| configuration, such as topics filtering
 and :term:`Participants <Participant>` configurations.
 
+.. _topic_filtering:
 
 Topic Filtering
 ===============
 
-.. note::
+|ddsrouter| allows filtering of DDS :term:`Topics<Topic>`, that is, it allows to define which DDS Topics are going to be
+relayed by the application.
+This way, it is possible to define a set of rules in |ddsrouter| to filter those data samples the user does not wish to
+forward. It is not mandatory to define such set of rules in the configuration file.
+In this case, a DDS Router will forward all the data published under the topics that it automatically discovers within
+the DDS network to which it connects.
```

Comment:
I will try to refactor these sentences.
First, separate the explanation about the configuration and the use of topic filtering.
Second, saying that data samples are filtered is incorrect, it should say DDS entities are created/activated in those topics.
Third, make it clearer that when a new topic is discovered in the network, Router will create entities to forward it if allowed.

Reply:
The use of these phrases without reference to the creation of internal entities is intended to be more understandable to the user. Therefore I would leave them as they are.